utils/src/main/kotlin/Extensions.kt

```diff
@@ -345,3 +346,15 @@ fun TypeFactory.constructTreeSetType(elementClass: Class<*>): CollectionType =
  * Check whether the URI has a fragment that looks like a VCS revision.
  */
 fun URI.hasRevisionFragment() = fragment?.let { Regex("[a-fA-F0-9]{7,}$").matches(it) } == true
+
+/**
+ * Return a version string with leading zeros of components stripped.
+ */
+fun stripLeadingZerosFromVersion(version: String) =
+    version.split(".").joinToString(".") {
+        try {
+            it.toInt().toString()
```

Comment:
Would e.g. ` "1.13.12".toInt()` actually yield an exception and if so would that be problematic for this function ?